fast_barcode_scanner/lib/src/barcode_camera.dart

```diff
@@ -96,11 +95,7 @@ class BarcodeCameraState extends State<BarcodeCamera> {
     if (!mounted) {
       return;
     }
-    if (cameraController.events.value != ScannerEvent.error && showingError) {
-      _safeSetState(() => showingError = false);
-    } else if (cameraController.events.value == ScannerEvent.error) {
-      _safeSetState(() => showingError = true);
-    }
+    _safeSetState(() {});
```

Comment:
I think we could just remove this block/line

Reply:
I'm not sure if didChangeDependencies will be updated other UI when it is called